runtime/queries/julia/injections.scm

```diff
@@ -1,5 +1,29 @@
-((triple_string) @injection.content
- (#set! injection.language "markdown"))
+(
+  (source_file
+    (
+      (string_literal) @injection.content
+      [
+        (module_definition)
+        (function_definition)
+        (macro_definition)
+        (primitive_definition)
+        (abstract_definition)
+        (struct_definition)
+        (assignment_expression)
+        (const_statement)
+      ]
+    ))
+  (#set! injection.language "markdown"))
```

Comment:
For this we'll want to use the anchor operator `.` to only match immediate siblings, or else the doc-string will apply to a function for example even if there's a for loop between the doc-string and function definition

```scm
(((string_literal) @injection.content
  .
  [
    (module_definition)
    (function_definition)
    (macro_definition)
    (primitive_definition)
    (abstract_definition)
    (struct_definition)
    (assignment_expression)
    (const_statement)
  ])
  (#set! injection.language "markdown"))
```

Reply:
Yeah, read this one wrong. Should be fixed now.